libs/interfaces/dist/libs/interfaces/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@baf-wallet/interfaces",
+  "version": "0.0.3",
+  "license": "MIT",
+  "main": "src/index.js",
+  "typings": "src/index.d.ts",
+  "dependencies": {
+    "near-api-js": "^0.39.0",
+    "buffer": "^5.7.1",
+    "@toruslabs/torus-direct-web-sdk": "^4.11.1"
+  },
+  "peerDependencies": {}
+}
```

Comment:
Can you add an exports property here so that we can import it like `@baf-wallet/core/interfaces` instead of having to do `@baf-wallet/core/src/interfaces` to get interfaces (and similarly for other non-default libs)?

see https://webpack.js.org/guides/package-exports/ for more information